src/components/selectAuthor/SelectAuthor.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './SelectAuthor.css';
+import Dropdown from 'react-bootstrap/Dropdown';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const SelectCategory = () => (
+
+  <Dropdown>
+    <Dropdown.Toggle variant="Info" id="lessonPanel">
+      <span id="titleCategory">Author</span>
+    </Dropdown.Toggle>
+
+    <Dropdown.Menu>
+      <Dropdown.Item href="#/action-1">Check status</Dropdown.Item>
+    </Dropdown.Menu>
+  </Dropdown>
+);
```

Comment:
- [ ] There should not be an author page, this should be ```category```  and should display only a button with the text "Check status"

Reply:
This is not an author page, this is the dropdown "selectCategory" that should be changed into author 
![book](https://user-images.githubusercontent.com/62058680/185668566-6347d6c2-0055-4c3f-ad4f-a806d26e6dfd.png)